menas/ui/css/style.css

```diff
@@ -36,33 +36,33 @@ html, body {
 
 .menasAddMappingFlex div {
     width: calc(100% - 38px);
-}
+  }
 
 .monitoringBlockLayoutCell {
   min-width: 20rem !important;
   max-width: 30rem !important;
 }
 
 .monitoringTitle {
-  text-align: Center;
+  textAlign: Center;
```

Comment:
I doubt that this is correct. https://www.w3schools.com/cssref/pr_text_text-align.ASP